libevmasm/Assembly.cpp

```diff
@@ -242,13 +245,153 @@ string Assembly::toStringInHex(u256 _value)
 	hexStr << std::uppercase << hex << _value;
 	return hexStr.str();
 }
+vector<Json::Value> Assembly::assemblyItemAsJSON(AssemblyItem const& _item, int _sourceIndex) const
+{
+	vector<Json::Value> result;
+
+	switch (_item.type())
+	{
+	case Operation:
+		result.emplace_back(createJsonValue(
+			instructionInfo(_item.instruction()).name,
+			_sourceIndex,
+			_item.m_modifierDepth,
+			_item.location().start,
+			_item.location().end,
```

Comment:
Can you find a solution where we don't have to pass there four arguments all the time to each of the function calls?

Reply:
True! That makes sense!